install_files/ansible-base/roles/app/tasks/initialize_securedrop_app.yml

```diff
@@ -114,3 +114,12 @@
     mode: 400
     backup: yes
   when: securedrop_header_image != "None"
+
+  # When a custom header image is not provided copy the example into the
+  # correct place
+- name: if a branded custom header image is NOT defined use the example logo
+  command: /bin/cp logo.png.example logo.png
+  args:
+    chdir: "{{ securedrop_code }}/static/i"
+    creates: "{{ securedrop_code }}/static/i/logo.png"
+  when: securedrop_header_image == "None"
```

Comment:
Minor note: I'd like it better if we used an empty string to indicate no custom header image file. `""` is not a valid filename, so it can't be confused with `"None"`, which is a valid filename. I'm pretty sure YAML does not map `"None"` to Python's `None`, so there's no reason to mimic that convention.